### PR TITLE
OCPBUGS-30160: Non-admin users are not able to see overview page

### PR DESCRIPTION
**Pull request by lokanandaprabhu**
**Issue**: https://issues.redhat.com/browse/OCPBUGS-30160

**Description**: As developer user, cluster version is not accessible from config.openshift.io resource. So added to fetch from window.SERVER_FLAGS

### CODE REVIEW

File: src/types/openshift.ts [hunk 1]
```diff
@@ -26,19 +26,3 @@ export type Release = {
   url?: string;
   channels?: string[];
 };
```
Comment (vikram-raj) [inline src/types/openshift.ts:28]: We can remove `UpdateHistory` and `Release` type definitions.

Comment (lokanandaprabhu) [reply src/types/openshift.ts]: Updated the PR

**Review by vikram-raj — COMMENTED** [review]
/lgtm

**Review by khrm — COMMENTED** [review]
/cherry-pick release-v1.14.x

**Pull request merged**